src/frontend/components/SøknadsSteg/Oppsummering/OppsummeringSteg/Eøs/EøsBarnOppsummering.tsx

```diff
@@ -14,25 +15,26 @@ import Oppsummeringsbolk from '../../Oppsummeringsbolk';
 import { StyledOppsummeringsFeltGruppe } from '../../OppsummeringsFeltGruppe';
 import EøsAndreForelderOppsummering from './EøsAndreForelderOppsummering';
 import EøsOmsorgspersonOppsummering from './EøsOmsorgspersonOppsummering';
-import { tittelSpmEøsBarnOppsummering } from './utils';
 
 interface Props {
     settFeilAnchors: React.Dispatch<React.SetStateAction<string[]>>;
     barn: IBarnMedISøknad;
-    nummer: string;
 }
 
-const EøsBarnOppsummering: React.FC<Props> = ({ settFeilAnchors, nummer, barn }) => {
+const EøsBarnOppsummering: React.FC<Props> = ({ settFeilAnchors, barn }) => {
```

Comment:
Tror det at man fjerner nummer knekker stegnummereringen av barna på oppsummeringssiden 🤔  Så her bør vi egentlig beholde det å legge det inn foran teksten til sanity eller noe sånt

Reply:
Her skulle det stått 5. og 6.
![Screenshot 2022-10-27 at 12 21 40](https://user-images.githubusercontent.com/8656966/198260089-9ef7faf5-0f27-49c8-b64e-2dd2713e7c3d.png)

Reply:
Ahh, det gir mening

Reply:
Fikset i [Hent riktig stegnummer for barn](https://github.com/navikt/familie-ks-soknad/pull/94/commits/cf4591672f63d9e9586537c28a7dad04ad51592a)
<img width="674" alt="image" src="https://user-images.githubusercontent.com/17828446/198270033-6adbd7d7-e28d-467d-99ee-5e43dfb1f528.png">